### ISSUE TITLE
Labels are not associated to combo boxes

### ISSUE THREAD

**Issue by claudiahub** [backstage/backstage#11264]
## Expected Behavior

Labels should be associated to combo boxes

## Actual Behavior

1. Navigate to `Home` on the left navigation. Make sure `Components` is selected on menu below the main heading.
2. Select `all` filter and in the table navigate to any application under name column with application live view plugin available (see screenshot below).
3. In the next page that opens up for selected appplication, please navigate to `Runtime resources` tab.
4. Navigate to `Resources` table and select the name of the application. A new page opens up.
5. Navigate to `Pods` table and select a link under name column.
6. In the new page that opens up, navigate to `Live view` section.
7. Select `Metrics` item from the menu item.
8. Verify whether labels are mapped with the comboboxes or not. 

<img width="1620" alt="image" src="https://user-images.githubusercontent.com/54085737/166501562-ebcd6047-f9a2-4d5b-a77c-f9c89870ac34.png">

## Context

Labels are not associated with the combo boxes.

Involved component(s):

- Select from '@backstage/core-components'

User Impact: 
Without the name of the combo box, it is difficult for screen reader users to know about the functionality of it. 
Thereby, they might not use the combo box.

A11y Conformance Level: A

WCAG Success Criteria: 1.3.1

## Environment

Browser Information: 
Google Chrome v100.0.4896.127
Firefox 99.0.1
Safari 15.4
